### DISCUSSION TITLE
Create components via CLI

### DISCUSSION THREAD

**Discussion by audreyteles** [wrabit/django-cotton#179] [Ideas]
Hi, i didn't use this library yet, but I'm thinking about to use soon.

I was reading the documentation and I think a feature to create components via CLI would be interesting, as it would avoid the possibility to creating components that don't follow the naming standards and help new users to get it about those standards... I think that is a good thing to have, but I don't know yet.

The idea would be similar to the example below:

**`cotton make:component my_component`**

Similar to the Laravel framework implementation. 

I just didn't find them in the documentation, so I'm not sure if this feature exists. But I want to contribute in some way.

**Comment by wrabit** [comment]
I have thought about it but I am choosing to spend time on features and stability improvements for now. Just remember, files are snake_case (mimicking python file naming), and reference them in templates as the kebab-case. (You can actually reference them as snake as well but it wont keep your html readable IMO)

**Reply by audreyteles** [reply]
I understand, I think the same thing, but I decided to discuss it first. 

Thank you for your attention.
